src/test/java/com/marizueva/laboratory/hw4/voidpageobject/pages/HomePage.java

```diff
@@ -0,0 +1,169 @@
+
+    @FindBy(css = ".icons-benefit.icon-practise")
+    WebElement displayedBenefitImage1;
+
+    @FindBy(xpath = "//*[@class='benefit-icon']/span[@class='icons-benefit icon-custom']")
+    WebElement displayedBenefitImage2;
+
+    @FindBy(xpath = "//*[@class='benefit-icon']/span[@class='icons-benefit icon-multi']")
+    WebElement displayedBenefitImage3;
+
+    @FindBy(xpath = "//*[@class='benefit-icon']/span[@class='icons-benefit icon-base']")
+    WebElement displayedBenefitImage4;
+
+    @FindBy(css = ".benefits> div:nth-child(1) span.benefit-txt")
+    WebElement benefitText1;
+
+    @FindBy(xpath = ".benefits> div:nth-child(2) span.benefit-txt")
+    WebElement benefitText2;
+
+    @FindBy(xpath = ".benefits> div:nth-child(3) span.benefit-txt")
+    WebElement benefitText3;
+
+    @FindBy(xpath = ".benefits> div:nth-child(4) span.benefit-txt")
+    WebElement benefitText4;
+
+    @FindBy(className = "benefit-txt")
+    private List<WebElement> textBelowImagesList;
+
+    @FindBy(xpath = "//div[@class='main-content']/h3[@class='main-title text-center']")
+    WebElement mainContentHeader;
+
+    @FindBy(xpath = "//div[@class='main-content']/p")
+    WebElement mainContentText;
+
+    @FindBy(xpath = "//iframe[@id='jdi-frame-site']")
+    WebElement iframeCenter;
+
+    @FindBy(xpath = "//img[@src='images/Logo_Epam_Color.svg']")
+    WebElement epamLogoInIframe;
+
+    @FindBy(xpath = "//div[@class='main-content']/h3[@class='text-center']")
+    WebElement subHeaderJdi;
+
+
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    @Step("Open site by URL  https://jdi-testing.github.io/jdi-light/index.html")
+    public void openSite() {
+        driver.navigate().to(siteUrl);
+    }
+
+    @Step("login default user: Roman Jdi1234")
+    public void performLogin(String userName, String password) {
+        openLoginDropdownButton.click();
+        loginField.sendKeys(userName);
+        passwordField.sendKeys(password);
+        enterCredentials.click();
+    }
+
+    @Step("Assert User name in the left-top side of screen that user is loggined 'ROMAN IOVLEV'")
+    public String getLoggedUserName() {
+        return loggedUserName.getText();
+    }
+
+    @Step("Assert that there are 4 items on the header section are displayed and\n"
+            + " they have proper texts\n"
+            + "  \"HOME\", \"CONTACT FORM\", \"SERVICE\", \"METALS & COLORS\"")
+    public String getHeaderItem(HeaderItems headerItem) {
+
+        switch (headerItem) {
+            case HOME:
+                return headerItemHome.getText();
+
+            case CONTACT_FORM:
+                return headerItemContact.getText();
+
+            case SERVICE:
+                return headerItemService.getText();
+
+            case METAL_AND_COLORS:
+                return headerItemMetalAndColor.getText();
+
+            default:
+                throw new NotFoundException("There is no such tab" + headerItem);
+
+        }
+    }
```

Comment:
Why is called assert? I do not see any assertions in this method?

Reply:
fixed